test-suites/gutenberg/sanity-test-suites.md

```diff
@@ -64,8 +64,8 @@ This holds a grouping of certain test suites to run in order to share the work w
 
 ### MediaText - 2
 
-- [ ] Media Text block - Insert video from device (TC002) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/media-text.md#tc002-v)
-- [ ] Media Text block - Insert image from device (TC002) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/media-text.md#tc002-i)
+- [ ] Media Text block - Insert video from device (TC002) - [TC002-v](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/media-text.md#tc002-v)
```

Comment:
I wonder if the `(TC002)` preceding the link is of any use  at this stage 🤔 . WDYT about removing it?

Reply:
(btw, same for the line right below this one)

Reply:
hmm yeah, I guess the sub task notation is enough i.e. `TC002-v`